content/en/continuous_integration/pipelines/gitlab.md

```diff
@@ -157,7 +158,7 @@ Fill in the integration configuration settings:
 **Default**: (empty, no additional tags)<br/>
 **Note**: Available only in GitLab.com and GitLab >= 14.8 self-hosted.
 
-You can test the integration with the **Test settings** button (only available when configuring the integration on a project). After it's successful, click **Save changes** to finish the integration set up.
+You can test the integration with the **Test settings** button (only available when configuring the integration on a project). After it's successful, click **Save changes** to finish the integration set up. Some customers have reported that the **Test settings** button sometimes fails. In this case, click **Save changes** and check that the first webhooks that are sent are successful by looking at the history in the "Recent events" section below.
```

Comment:
What is the "Recent events" section? Is this a reference to a section on this page?

```suggestion
You can test the integration with the **Test settings** button (only available when configuring the integration on a project). After it's successful, click **Save changes** to finish the integration set up. If the button fails, click **Save changes** and check that the first webhooks sent are successful by looking at the history in the "Recent events" section below.
```

Reply:
Yes, on the same page there is a "Recent events" section
![image](https://github.com/user-attachments/assets/a303bdc1-895d-4e95-a967-99b4302fbe9a)